requirements-dev.txt

```diff
@@ -8,3 +9,5 @@ lxml
 mock
 pytest
 requests-mock
+
+git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.2.0#egg=digitalmarketplace-test-utilts==2.2.0
```

Comment:
Just trying this out locally - I think there's a typo here: `test-utilts`

Reply:
Nice spot